chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,786 @@
+patients with worse health conditions might not survive to follow-up,
+and so on.
+If this is the case, then your results might only be an effect of your remaining sample
+being a subset of the original sample that were better or worse off from the beginning.
+You should have a variable in your master dataset that indicates attrition.
+A balance check using the attrition variable can provide insights
+as to whether the lost observations were systematically different
+compared to the rest of the sample,
+and there are a variety of methods for estimating treatment effects
+with selective attrition.
+
+%-----------------------------------------------------------------------------------------------
+\subsection{Monitoring data}
+
+For any study with an ex-ante design, 
+\textbf{monitoring data}\sidenote{\url{
+		https://dimewiki.worldbank.org/Monitoring\_Data}}
+is very important for understanding whether field realities match the research design.
+\index{monitoring data}
+Monitoring data is used to understand if the
+assumptions made during the research design corresponds to what is true in reality.
+The most typical example is to make sure that, in an experimental design,
+the treatment was implemented according to your treatment assignment.
+
+The implementation of your treatment often have to be carried by field counterparts
+that have more complex realities than only your research.
+Field work will always be more intricate than a lab environment, 
+and the field counterpart might not be aware that their actions are the implementation of your research.
+Therefore, you must acquire monitoring data that tells you how well the treatment assignment in the field
+corresponds to your intended treatment assignment,
+for nearly all research designs.
+
+Another example of a research design where monitoring data is important
+are regression discontinuity (RD) designs
+where the discontinuity is a cutoff for eligibility of the treatment. 
+For example, 
+let's say your project studies the impact of a program for students that scored under 50\% at a test.
+We might have the exact results of the tests for all students, 
+and therefore know who should be offered the program, 
+however that is not the same as knowing who attended the program. 
+A teacher might offer the program to someone that scored 51\% at the test,
+and someone that scored 49\% at the might decline to participate in the program.
+We should not pass judgment on a teacher that offers a program to a student
+they think can benefit from it, 
+but if that was not inline with our research assumptions,
+then we need to understand how common that was.
+Otherwise the result of our research will not be helpful
+in evaluating the program.
+
+Monitoring data is particularly prone to errors 
+relating to merging with other data set. 
+It is too late to think about how monitoring data will be merged
+with the rest of the data, after the monitoring data is already collected.
```

Comment:
well, you still need to merge it after it is collected, so it's not too late for that.
I assume the idea is that it is too late to think about what it should look like to be most efficiently merged?

Reply:
I see your point. Or that it is the least error prone to do it before hand.

Reply:
Simply removed that sentence in 0c76937f6f91febac870f266c83dc337d22f47d9. Two sentences later we talk about the best time to figure this out when you prepare data map.